composer.json

```
@@ -3,13 +3,12 @@
     "type": "symfony-bundle",
     "description" : "Provide APCu support",
     "require": {
-        "php"                            : ">=5.5",
+        "php"                            : ">=7.2",
         "ext-apcu"                       : ">=4.0.7, <5.3"
     },
     "require-dev": {
-        "atoum/atoum"                    : "^2.8||^3.0",
-        "m6web/coke"                     : "~1.2",
-        "m6web/symfony2-coding-standard" : "~1.1"
+        "atoum/atoum"                    : "^3.4||^4.0",
```

```suggestion
        "atoum/atoum"                    : "^4.0",
```

if we drop `< php7.4`